docs/example_figure.py

```diff
@@ -0,0 +1,86 @@
+from matplotlib import pyplot as plt
+from matplotlib import colors as colors
+import numpy as np
+from astropy import units as u
+from screens.fields import dynamic_field
+from screens.visualization import make_sketch
+
+
+def gen_magnifs(th):
+
+    # Complex magnifications amplitudes of the scattering points
+    sig = 1.5*u.mas
+    a = 0.3*np.exp(-0.5*(th/sig)**2).to_value(1)
+    np.random.seed(2)
+    magnifs = a * np.random.normal(size=(th.size, 2)).view('c16').squeeze(-1)
+    magnifs[np.where(th == 0)] = 1
+    magnifs /= np.sqrt((np.abs(magnifs)**2).sum())
+
+    return magnifs
+
+
+def example_figs(th_par, magnifs,
+                 d_eff=2.*u.kpc, mu_eff=50.*u.mas/u.yr, fobs=330.*u.MHz,
+                 delta_f=1.*u.MHz, delta_t=20.*u.minute, nf=201, nt=101,
+                 **kwargs):
+
+    # Observation frequencies and times
+    f = fobs + np.linspace(-0.5*delta_f, 0.5*delta_f, nf)
+    t = np.linspace(-0.5*delta_t, 0.5*delta_t, nt)[:, np.newaxis]
+
+    # Create dynamic wavefield and dynamic spectrum
+    th_perp = np.zeros_like(th_par)
+    dynwave = dynamic_field(th_par, th_perp, magnifs, d_eff, mu_eff, f, t)
+    axes = tuple(range(0, dynwave.ndim-2))
+    dynwave = dynwave[...].sum(axes)
+    dynspec = np.abs(dynwave)**2
+
+    # Make conjugate and secondary spectrum
```

Comment:
More general: I wonder if it would make sense to just use `SecSpec.from_dynspec`?

Reply:
I just tried this and the resulting secondary spectrum seems much brighter (more power in the points other than the origin). Is this secondary spectrum normalized to the power at the origin? Maybe I did something very wrong when producing the conjugate / secondary spectrum.

Reply:
Yes, it should be normalized at the origin. But did you at the same time remove the padding?  That padding does reduce the brightness (since you just add a lot of pixels with just mean value).

Reply:
I got confused because `SecondarySpectrum.secspec` seems to be (what I thought is referred to as) the conjugate spectrum.

Reply:
Hmm, yes, I think my nomenclature is bad here! Maybe we should rename the class...